vendor.conf

```diff
@@ -1,13 +1,14 @@
 github.com/BurntSushi/toml master
 github.com/Nvveen/Gotty master
 github.com/blang/semver master
-github.com/containers/image 106607808da3cff168be56821e994611c919d283
-github.com/containers/storage 5d8c2f87387fa5be9fa526ae39fbd79b8bdf27be
+github.com/containers/image 063852766c3e82ec8359ce5f6612e056f3efaa76
+github.com/containers/storage 43c477703fe73129e2186cf730601c52d309c3ef
 github.com/docker/distribution master
-github.com/docker/docker 0f9ec7e47072b0c2e954b5b821bde5c1fe81bfa7
+github.com/docker/docker master
```

Comment:
Try pinning this to revision 30eb4d8cdc422b023d5f11f29a82ecb73554183b, which is the version that `containers/image` lists in its `vendor.conf`.